airbyte-webapp/src/hooks/services/Analytics/useAnalyticsService.tsx

```diff
@@ -76,6 +76,17 @@ export const useAnalyticsIdentifyUser = (userId?: string, traits?: Record<string
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [userId]);
 };
+export const useAnalyticsSetAnonymousIdFromLocalStorage = (): void => {
+  const analyticsService = useAnalyticsService();
+  const anonymousId = localStorage.getItem("ajs_anonymous_id");
+
+  useEffect(() => {
+    if (anonymousId) {
+      analyticsService.setAnonymousId(JSON.parse(anonymousId));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We'd usually just add the `analyticsService` here as well. It's a stable reference so it won't cause us problems with rerendering, but helps not needing to disable eslint rules here.